source/elements/tags/Alert/Alert.jsx

```diff
@@ -0,0 +1,42 @@
+export const Alert = (props) => {
+  const {
+    tagName: Tag,
+    className,
+    variant,
+    children,
+    ...attrs
+  } = props;
+
+  const classStack = FcUtils.createClassStack([
+    'Alert',
+    `Alert--${variant}`,
+    className
+  ]);
+
+  return (
+    <Tag className={classStack} {...attrs}>
+      <div className="Alert-body">
```

Comment:
BEM stylings dictate this should be `Alert__body` instead of `Alert-body`. It also looks like you aren't adding any styles to this div. Is it necessary to include?